src/common/containers/EditorRecordActionContainer.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Set } from 'immutable';
+import { doSetsHaveCommonItem } from '../utils';
+import ListItemAction from '../../common/components/ListItemAction';
+
+const ONLY_SUPER_USERS_AND_CATALOGERS = Set(['superuser', 'cataloger']);
+
+class EditorRecordActionContainer extends Component {
+  render() {
+    const { userRoles, recordId } = this.props;
+    const isAuthorized = doSetsHaveCommonItem(
+      userRoles,
+      ONLY_SUPER_USERS_AND_CATALOGERS
+    );
+    const href = `/workflows/edit_article/${recordId}`;
+    if (isAuthorized) {
+      return (
+        <ListItemAction
+          iconType="edit"
+          text="Edit"
```

Comment:
Change the text to 'edit' so that is consistent with the other lowercase buttons. Sorry, I now realise that in the mockup I had it in capital, but it's better to have consistency